<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrcNetworkWidget</class>
 <widget class="QWidget" name="IrcNetworkWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTextBrowser" name="messages">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Shows the network messages</string>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="networkCombo">
       <property name="toolTip">
        <string>Select a network to connect to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="connectButton">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="awayButton">
       <property name="toolTip">
        <string>Press to set the user status to AWAY</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editButton">
       <property name="toolTip">
        <string>Press to edit the networks</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="nickCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select a nick name for the channel</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="channelCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enter the channel to join</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="joinButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Press to join the channel</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>networkCombo</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>awayButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>nickCombo</tabstop>
  <tabstop>channelCombo</tabstop>
  <tabstop>joinButton</tabstop>
  <tabstop>messages</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
